docs/prepare-your-dev-environment.md

```diff
@@ -21,54 +18,53 @@ This document goes through the development dependencies one requires in order to
 1. Install [Podman](https://podman.io/getting-started/installation) and [podman-docker](https://developers.redhat.com/blog/2019/02/21/podman-and-buildah-for-docker-users#) if you haven't already, used for building container images.
 
 1. Run for `az acr login` compatability
-```bash
-sudo touch /etc/containers/nodocker
-```
+    ```bash
+    sudo touch /etc/containers/nodocker
+    ```
 
 ### Fedora Packages
 1. Install the `gpgme-devel`, `libassuan-devel`, and `openssl` packages.
-> `sudo dnf install -y gpgme-devel libassuan-devel openssl`
+    > `sudo dnf install -y gpgme-devel libassuan-devel openssl`
 
 1. Install [Docker 17.05+](https://docs.docker.com/engine/install/fedora/) or later, used as an alternative to podman.
 
 ### Debian Packages
-1. Install the `libgpgme-dev` package.
+
+Install the `libgpgme-dev` package.
 
 ### MacOS Packages
+
 1. We are open to developers on MacOS working on this repository.  We are asking MacOS users to setup GNU utils on their machines.
 
-We are aiming to limit the amount of shell scripting, etc. in the repository, installing the GNU utils on MacOS will minimise the chances of unexpected differences in command line flags, usages, etc., and make it easier for everyone to ensure compatibility down the line.
+    We are aiming to limit the amount of shell scripting, etc. in the repository, installing the GNU utils on MacOS will minimise the chances of unexpected differences in command line flags, usages, etc., and make it easier for everyone to ensure compatibility down the line.
+
+    Install the following packages on MacOS:
+    ```bash
+    # GNU Utils
+    brew install coreutils findutils gnu-tar grep
 
-Install the following packages on MacOS:
-```bash
-# GNU Utils
-brew install coreutils
-brew install findutils
-brew install gnu-tar
-brew install grep
+    # Install envsubst (provided with gettext)
+    brew install gettext
+    brew link gettext
 
-# Install envsubst
-brew install gettext
-brew link --force gettext
+    # Install gpgme
+    brew install gpgme
+    ```
 
-# Install
-brew install gpgme
+1. Modify your `~/.zshrc` (or `~/.bashrc` for Bash): this prepends `PATH` with GNU Utils paths;
 
-# GNU utils
-# Ref: https://web.archive.org/web/20190704110904/https://www.topbug.net/blog/2013/04/14/install-and-use-gnu-command-line-tools-in-mac-os-x
-# gawk, diffutils, gzip, screen, watch, git, rsync, wdiff
-export PATH="/usr/local/bin:$PATH"
-# coreutils
-export PATH="/usr/local/opt/coreutils/libexec/gnubin:$PATH"
```

Comment:
On ARM version of macOS, homebrew installs everything into `/opt/homebrew` instead of multiple places (`/usr/local/bin`, `/usr/local/opt`, etc).

`brew --prefix` shows the right location of the installation.